rest-api/test/unit_test/dao_test/biobank_order_dao_test.py

```diff
@@ -319,3 +319,5 @@ def test_amending_an_order(self):
     self.assertEqual(amended_order.version, 2)
     self.assertEqual(amended_order.orderStatus, BiobankOrderStatus.AMENDED)
     self.assertEqual(amended_order.amendedReason, 'I had to change something')
+    self.assertEqual(amended_order.amendedSiteId, 1)
+    self.assertEqual(amended_order.amendedUsername, 'mike@pmi-ops.org')
```

Comment:
improve biobank_order_dao_test to make sure amended site and username are updated.